include/lexy/parse_tree.hpp

```diff
@@ -132,7 +132,7 @@ struct pt_node_token : pt_node<Reader>
         if constexpr (_optimize_end)
         {
             static_assert(!std::is_pointer_v<typename Reader::iterator>
-                          || sizeof(pt_node_token) == 3 * sizeof(void*));
+                          || sizeof(pt_node_token) == 2 * sizeof(void*) + 8);
```

Comment:
I saw some fixed size things and so updated this static assertion to let the 32-bit build compile, but it needs double-checking really.